Form-Controls/myform.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+
+        <title>My form exercise</title>
+
+    </head>
+    <body>
+        <header>
+            <h1>Product Pick</h1>
+        </header>
+
+        <main>
+
+          <form>
+            <fieldset>
+                <legend>Customer Details</legend>
+                <p>
+                    <label for="name">Name:</label>
+                    <input type="text" id="name" name="user_name" minlength="2" required />
+                </p>
+                <p>
+                    <label for="email">Email:</label>
+                    <input type="email" id="email" name="user_email" required />
+                </p>
+            </fieldset>
+            <fieldset>
+                <legend>T-Shirts Information</legend>
+                <strong>Colour</strong>
+                <p>
+                    <input type="radio" name="T_colour" id="colour" value="Blue" required />
```

Comment:
For html attributes, please, use lowercase letters and hyphens only, this is the best practice in the industry.